data/mods/megasforall/abilities.ts

```diff
@@ -1672,38 +1664,18 @@ export const Abilities: {[abilityid: string]: ModdedAbilityData} = {
 		num: -1047,
 	},
 	bodyofwater: {
-		desc: "When this Pokémon uses a Water-type attack, damage is calculated using the user's Defense stat as its Attack or its Special Defense as its Special Attack, including stat stage changes. Other effects that modify the Attack and Special Attack stats are used as normal.",
+		desc: "When this Pokémon uses a Water-type attack, damage is calculated using the user's Defense stat as its Attack or its Special Defense as its Special Attack. Other effects that modify the Attack and Special Attack stats are used as normal, including stat stage changes.",
 		shortDesc: "Water-type attacks use Def as Atk and Sp. Def as Sp. Atk in damage calculation.",
 		name: "Body of Water",
 		onModifyMove(move, attacker) {
 			if (move.type === 'Water') {
 				move.useSourceDefensiveAsOffensive = true;
+				(move as any).bodyofwaterBoosted = true;
```

Comment:
Does it make more sense to have an `abilityBoosts: {[k: string]: boolean}` property on moves? That would eliminate the need for all these `as any` casts, since it could go on the main `Move` type definition. I don't know how many other OMs use this, though—it's probably not worth it for a single format.

Reply:
i dont think any other OMs use this, but it'd be useful for pixilate/refrigerate etc. probably worth looking into at a later date